docs/docs/supply-chain/vex.md

```diff
@@ -470,6 +474,100 @@ does not match:
 - `pkg:maven/com.google.guava/guava@24.1.1?classifier=sources`
     - `classifier` must have the same value.
 
+Here is a proposed addition to the Trivy VEX documentation, focusing on how VEX is applied to dependency graphs:
+
+### Applying VEX to Dependency Trees
+
+Trivy internally generates a dependency tree and applies VEX statements to this graph.
```

Comment:
maybe link to the dependency tree docs for more info?
https://aquasecurity.github.io/trivy/v0.52/docs/configuration/reporting/#show-origins-of-vulnerable-dependencies

Reply:
I think this can be confusing.
This logic is not related to the dependency tree display.